### PR TITLE
Add EVA at Sarek pipeline institutional profile

### PR DESCRIPTION
**Pull request by jfy133**
---
name: New Config
about: A new cluster config
---

Please follow these steps before submitting your PR:

- [ ] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [ ] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [x] Add your custom config file to the `conf/` directory
- [x] Add your documentation file to the `docs/` directory
- [x] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [x] Add your custom profile to the `README.md` file in the top-level directory
- [ ] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]
lots of JAVA process I see

File: conf/pipeline/sarek/eva.config [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+env {
+    _JAVA_OPTIONS = "-XX:ParallelGCThreads=1"
+    OPENBLAS_NUM_THREADS = 1
+    OMP_NUM_THREADS = 1
+}
+
+process {
+  withName:GATK4_APPLYBQSR {
```
Comment (FriederikeHanssen) [inline conf/pipeline/sarek/eva.config:17]: What about fastqc/snpeff and fgbio?

Comment (jfy133) [reply conf/pipeline/sarek/eva.config:17]: Will update as necessary, FASTQC 'worked' on the test run with no issue strangely...

Comment (jfy133) [reply conf/pipeline/sarek/eva.config:17]: i.e., I've asked the people who want to use the pipeline what tools htey will use from the list :)

**Pull request merged**